x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -372,7 +344,7 @@ func (k Keeper) chargeComputeUnitsToProjectAndSubscription(ctx sdk.Context, clie
 		return fmt.Errorf("failed to add CU to the subscription")
 	}
 
-	err = k.subscriptionKeeper.AddTrackedCu(ctx, sub.Consumer, relay.Provider, relay.SpecId, cuAfterQos, sub.Block)
+	err = k.subscriptionKeeper.AddTrackedCu(ctx, sub.Consumer, relay.Provider, relay.SpecId, cuAfterQos)
```

Comment:
you removed the block, and the block is necessary, you can;t have only one entry of trackedCU, if a month changed, and you add cu for an epoch form the previous month you add it to the subscription that was ended, but if it's based on the new month you add it to the new subscription tracked CU

Reply:
done. see https://github.com/lavanet/lava/commit/69856dfd2c652fe9a99b8d12ac26d589e909f5d0